lib/api/tenure/v1/service.ts

```diff
@@ -83,13 +83,14 @@ export const editTenure = async ({ id, ...data }: EditTenureParams): Promise<voi
   return response.data;
 };
 
-export const editTenuredAsset = async ({
-  id,
-  ...data
-}: EditTenuredAssetParams): Promise<void> => {
-  const response = await axiosInstance.patch(
-    `${config.tenureApiUrlV1}/tenures/${id}`,
-    data,
-  );
-  return response.data;
+export const editTenuredAsset = async (
+  id: string,
+  tenuredAssetUpdate: EditTenuredAssetParams,
+) => {
+  return new Promise<void>((resolve, reject) => {
+    axiosInstance
+      .patch(`${config.tenureApiUrlV1}/tenures/${id}`, tenuredAssetUpdate, {})
```

Comment:
I can't see editTenuredAsset being referenced in the code anywhere, so I'm not sure what is in ...data, but I can see EditTenuredAssetParams (from the screenshot in the other comment) has an etag property so that should be fine.

Lastly why is there an empty object sent at the end of line 92?